hbase-server/src/main/java/org/apache/hadoop/hbase/master/ClusterSchema.java

```diff
@@ -82,8 +82,7 @@
    * @param nonceKey A unique identifier for this operation from the client or process.
    * @param latch A latch to block on for precondition validation
    * @return procedure id
-   * @throws IOException Throws {@link ClusterSchemaException} and {@link InterruptedIOException}
-   *    as well as {@link IOException}
+   * @throws IOException
```

Comment:
Just remove the throws if no extra javadoc.

Reply:
I think removal of `@throws` might cause javadoc warnings, providing some description here would be better. Based on impl, it seems we might not be able to get rid of `throws` from method signature.

Reply:
OK, i will add some description about the exception, thanks